tests/rules/flowbits/test.rules

```diff
@@ -0,0 +1,8 @@
+alert ip any any -> any any (msg:"Flowbit noalert"; flowbits:noalert,fb1; sid:1;)
+alert ip any any -> any any (msg:"Flowbit isset"; flowbits:isset,fb1; sid:2;)
+alert ip any any -> any any (msg:"Flowbit isnotset"; flowbits:isnotset,fb1; sid:3;)
+alert ip any any -> any any (msg:"Flowbit set"; flowbits:set,fb1; sid:4;)
+alert ip any any -> any any (msg:"Flowbit unset"; flowbits:unset,fb1; sid:5;)
+alert ip any any -> any any (msg:"Flowbit toggle"; flowbits:toggle,fb1; sid:6;)
+alert ip any any -> any any (msg:"Flowbit isset ored flowbits"; flowbits:isset,fb1|fb2; sid:7;)
+alert ip any any -> any any (msg:"Flowbit isnotset ored flowbits"; flowbits:isnotset, fb1|fb2 ; sid:8;)
```

Comment:
As I've explained in the Suricata PR, the ored flowbits variables won't exist, nor be logged, if they have not been set with `flowbits:set` before. So, `fb2` won't exist for the engine.

Reply:
Oh so I should create another rule for fb2 as such and then use it in isset and isnotset rules:
`alert ip any any -> any any (msg:"Flowbit set"; flowbits:set,fb2; sid:2;)`
Am I correct to assume this and if so, should I also create duplicates of all other rules for fb2?

Reply:
Yes, you should create a rule for setting fb2. But there's no need to create `isset` or `isnoset` rules for it. It `set` will setup the variable name with Suri. :)

Reply:
Not, there's no need.